### ISSUE TITLE
Tooltip aria-label is undefined

### ISSUE THREAD

**Issue by mattalco** [carbon-design-system/carbon#4224]
## Environment

> macOS

> Chrome

> pa11y

> Chrome dev tools

## Detailed description (see screenshots)

> 10.6.0 (edit)

> I expected, according to the source code, that the Tooltip would have an aria-label defined by default

> The aria-label is undefined

> What WCAG 2.1 checkpoint does the issue violate? Not sure

## Steps to reproduce the issue

1. Create a Tooltip react component on any page
2. Verify aria-label doesn't exist

> Please create a reduced test case in CodeSandbox
> http://react.carbondesignsystem.com/?path=/story/tooltip--default-bottom

## Additional information
Tooltip.js
<img width="557" alt="Screen Shot 2019-10-04 at 10 39 28 AM" src="https://user-images.githubusercontent.com/13972013/66216489-8eb02400-e693-11e9-961f-4cc92cccb9d9.png">

Chrome Dev Tools inspecting the DOM
<img width="487" alt="Screen Shot 2019-10-04 at 10 41 11 AM" src="https://user-images.githubusercontent.com/13972013/66216490-8eb02400-e693-11e9-9a36-d2b601c32373.png">

**Comment by jendowns** [comment]
@mattalco Do you have a valid `iconDescription` prop defined?
The `aria-label` attribute will only be added if you *don't* have `triggerText` & if you *do* provide a valid `iconDescription`. See this propType documentation: https://github.com/carbon-design-system/carbon/blob/master/packages/react/src/components/Tooltip/Tooltip.js#L166

**Comment by mattalco** [comment]
That worked! Sorry missed that in the code/StoryBook. Thanks for pointing that out to me.
